golang/cosmos/x/swingset/config.go

```diff
@@ -27,30 +45,74 @@ const DefaultConfigTemplate = `
 ###############################################################################
 
 [swingset]
-# slogfile is the path at which a SwingSet log "slog" file should be written.
+# The path at which a SwingSet log "slog" file should be written.
 # If relative, it is interpreted against the application home directory
 # (e.g., ~/.agoric).
+# May be overridden by a SLOGFILE environment variable, which if relative is
+# interpreted against the working directory.
 slogfile = "{{ .Swingset.SlogFile }}"
 
 # The maximum number of vats that the SwingSet kernel will bring online. A lower number
 # requires less memory but may have a negative performance impact if vats need to
 # be frequently paged out to remain under this limit.
-max_vats_online = {{ .Swingset.MaxVatsOnline }}
+max-vats-online = {{ .Swingset.MaxVatsOnline }}
+
+# Retention of vat snapshots, with values analogous to those of export
+# ` + "`artifactMode`" + ` (cf.
```

Comment:
Is this dance with the quotes and plusses only there so to get literal backticks into the sample config file? I'd have been content with single quotes.

Reply:
oh, or maybe the generated config file is embedding some computed values, from defaults established elsewhere, so `artifactMode` is actually a variable.. nevermind